Makefile

```diff
@@ -17,4 +17,4 @@
 ARROW := \033[34;1m=>\033[0m
 
 # order matters for these
```

Comment:
For reviewers:

- db.sql.text is the best place to start to understand the schema. This works the same way as our existing data store implementations, but it's using SQL. 
- postgresclient.go has the actual implementation.
- postgresclient_test.go is where we hook up the existing tests.